routers/web/repo/projects.go

```diff
@@ -236,6 +240,10 @@ func RenderEditProject(ctx *context.Context) {
 	ctx.Data["card_type"] = p.CardType
 	ctx.Data["redirect"] = ctx.FormString("redirect")
 	ctx.Data["CancelLink"] = fmt.Sprintf("%s/projects/%d", ctx.Repo.Repository.Link(), p.ID)
+	ctx.Data["ProjectMarkdownPreviewURL"] = fmt.Sprintf("%s/markup", ctx.Repo.Repository.Link())
+	ctx.Data["ProjectMarkdownPreviewContext"] = ctx.Repo.Repository.Link()
+	ctx.Data["ProjectMarkdownPreviewMode"] = "comment"
+	ctx.Data["ProjectMarkdownHideRepoButtons"] = false
```

Comment:
Maybe these data values are not necessary in Go code, because "projects/new.tmpl" clearly knows it is working as "mode=comment", etc.

Reply:
This template is used by projects on a repo and projects on a user level, so different modes need to be used.

Reply:
Why they need different modes? What's the difference?

Reply:
I think we should render the issue reference everywhere in the Repo. A new mode could be useful, but that's out of scope for this PR.

Reply:
But the code is using `Metas: map[string]string{"mode": "document"},` ?